src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -89,6 +89,7 @@ public class AzureSynapsePdao {
   private static final String DEFAULT_CSV_QUOTE = "\"";
   private static final String EMPTY_TABLE_ERROR_MESSAGE =
       "Unable to query the parquet file for this table. This is most likely because the table is empty.  See exception details if this does not appear to be the case.";
+  private static final String MAX_BIG_INT = "9223372036854770000";
```

Comment:
Isn't this `String.valueOf(Long.MAX_VALUE)`? Hmm actually it isn't since Java max long is `9223372036854775807`??

Reply:
Well that's weird. I wonder what they did with the extra bytes? https://docs.informatica.com/integration-cloud/data-integration-connectors/current-version/microsoft-azure-synapse-sql-connector/data-type-reference/microsoft-azure-synapse-sql-and-transformation-data-types.html

Reply:
Yeah, I just pulled the number from the Synapse docs (and left it as a string so I didn't need to worry about number limits in Java)